queries.sql

```diff
@@ -233,3 +233,9 @@ JOIN species ON species.id = animals.species_id
 GROUP BY species.name
 ORDER BY COUNT(visits.id)
 DESC LIMIT 1;
+
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits WHERE animal_id = 4;
+
+EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;
```

Comment:
- [ ] [REQUIRED] Good job so far! The second screenshot should show how much time the query takes `AFTER` the improvements are introduced by you
- The [execution time should be significantly lower](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md) than `1 sec (1000 ms)` . Please after implementation this request change the `second` screenshot result 🙏🏼 

![image](https://github.com/mwismann/vet_clinic_2/assets/63915024/abad0d98-9cb7-4a5f-bcfd-e14970ba6c52)
![image](https://github.com/mwismann/vet_clinic_2/assets/63915024/4259be42-5eda-4528-903f-7c9d23ed11aa)

Reply:
Hi, @mukhammadrizooff! I hope you are doing well 🤠

Regarding the project's requirements found at [this link](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md), the "execution time: X ms and X < 1000" metric pertains to the time it takes to analyze a query after inserting a specific amount of data. It's important to note that this measurement doesn't represent the expected execution time after improvements. Instead, it should be greater than 1000 ms when first inserting data before any enhancements.

If you take a look at our screenshot, you'll see that after inserting data into the database multiple times, we observed an execution time of 5600 ms, which met the initial requirement for proceeding with the project.

Once the improvements were implemented, we reanalyzed the query and achieved an outstanding execution time of 1968 ms, a significant optimization compared to the initial execution time.